phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1685,12 +1685,16 @@ public MutationState createIndex(CreateIndexStatement statement, byte[][] splits
                 for (ColumnName colName : requiredCols) {
                     // acquire the mutex using the global physical table name to
                     // prevent this column from being dropped while the view is being created
+                    String colNameSeparatedByDot = SchemaUtil
+                            .replaceColonToDot(colName.getColumnName());
```

Comment:
This seem preferred way based on existing usages:
```
String colNameSeparatedByDot = colName.getColumnName().replace(QueryConstants.NAMESPACE_SEPARATOR, QueryConstants.NAME_SEPARATOR)
```

Reply:
Okey I'll use it, Thanks